src/tests/components/ProjectForm.test.js

```diff
@@ -21,4 +23,50 @@ describe('ProjectForm', () => {
     form.simulate('submit')
     expect(props.handleSubmit).toHaveBeenCalledTimes(1)
   })
+
+  it('adds new repo fields', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <ConnectedProjectForm />
+      </Provider>
+    )
+    wrapper.find('.ui.button.field-group__add').first().simulate('click')
+    expect(wrapper.find('Field[name="repos[0].value"]').length).toBeTruthy()
```

Comment:
I am not a fan of `.toBeTruthy()` tests cause I feel like they are flaky... is there a way that you can test this without using it? 
here is something on the subject, if you want to have a look https://stackoverflow.com/questions/32615713/tobetrue-vs-tobetruthy-vs-tobetrue

I know there is `exists()` https://airbnb.io/enzyme/docs/api/ReactWrapper/exists.html... 
I'm assuming it doesn't have a predetermined length that we could use `toHaveLength(n)`?